### PR TITLE
Set default initiative status filter to closed

### PR DESCRIPTION
**Pull request by jarviiis21**
#### Solved task: 

> Set default initiative status filter to closed in initiatives list.

### CODE REVIEW

File: app/decorators/decidim/initiatives/initiatives_controller_decorator.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@
   # Assigns the value "closed" to the :state key until the given date is reached.
   def default_filter_params
     original_default_filter_params.tap do |default_params|
-      default_params[:state] = "closed" if Time.current < SIGNATURE_START_DATE
+      default_params[:state] = "closed"
```
Comment (tramuntanal) [inline app/decorators/decidim/initiatives/initiatives_controller_decorator.rb]: The requested feature is to show "filed" initiatives after the `SIGNATURE_START_DATE` is reached.
Thus this should not be the default state.

**Comment by tramuntanal** [comment]
hi @victorol1 can you reprend this PR please?

File: app/decorators/decidim/initiatives/initiatives_controller_decorator.rb (hunk 1 above)
Comment (victorol1) [reply app/decorators/decidim/initiatives/initiatives_controller_decorator.rb]: Done!

**Comment by victorol1** [comment]
@tramuntanal changes applied, pending review

File: app/decorators/decidim/initiatives/initiatives_controller_decorator.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@
   # Assigns the value "closed" to the :state key until the given date is reached.
   def default_filter_params
     original_default_filter_params.tap do |default_params|
-      default_params[:state] = "closed" if Time.current < SIGNATURE_START_DATE
+      default_params[:state] = Time.current < SIGNATURE_START_DATE ? "open" : "closed"
```
Comment (tramuntanal) [inline app/decorators/decidim/initiatives/initiatives_controller_decorator.rb]: Shouldn't it be the oher way around?
```suggestion
      default_params[:state] = Time.current < SIGNATURE_START_DATE ? "closed" : "open"
```

Comment (victorol1) [reply app/decorators/decidim/initiatives/initiatives_controller_decorator.rb]: **No, in production the default state right now is "open" and they want "closed" to be shown instead.**
imagen

**Pull request merged**